src/components/shop-dialog.tsx

```diff
@@ -64,20 +133,20 @@ export default function ShopDialog() {
                   setIsLoadingBuy(false);
                 }}
               >
-                <div className="flex items-center">
-                  {/* <img
-                    src={`/images/${item.type}.png`}
-                    alt={item.type}
-                    className="w-8 h-8 mr-2"
-                  /> */}
-                  <span>{item.type}</span>
+                {getImageByItemType(item.type)}
+                <div className="grow">
+                  <FontAwesomeIcon
+                    icon={faGem}
+                    size="lg"
+                    className="text-blue-400"
+                  />
+                  <span className="ml-1">
+                    {new Intl.NumberFormat(locale).format(item.cost)}
+                  </span>
                 </div>
-                <FontAwesomeIcon
-                  icon={faGem}
-                  size="lg"
-                  className="mt-2 cursor-pointer text-blue-400"
-                />{" "}
-                <span>{new Intl.NumberFormat(locale).format(item.cost)}</span>
+                <button className="cursor-pointer rounded bg-green-600 px-4 py-2 font-bold text-white">
```

Comment:
Please use the `Button` component from shadcn and use it with as little as possible extra styles.